wrapanapi/systems/redfish.py

```diff
@@ -178,7 +252,57 @@ def get_server(self, resource_id):
         """
         return RedfishServer(self, raw=self.api_client.find(resource_id))
 
+    def get_chassis(self, resource_id, required_types=None):
```

Comment:
Maybe it would be a bit nicer for the method callers to define it as
```suggestion
    def get_chassis(self, resource_id, *required_types):
```

This will remove the need for adding `[]` around the last group of parameters.

Reply:
👍